x-pack/plugins/fleet/server/services/preconfiguration.ts

```diff
@@ -45,7 +45,7 @@ export async function ensurePreconfiguredPackagesAndPolicies(
     // If there are multiple packages with duplicate versions, separate them with semicolons, e.g
     // package-a:1.0.0, package-a:2.0.0; package-b:1.0.0, package-b:2.0.0
     const duplicateList = duplicatePackages
-      .map(([, versions]) => versions.map((v) => `${v.name}:${v.version}`).join(', '))
+      .map(([, versions]) => versions.map((v) => `${v.name}-${v.version}`).join(', '))
```

Comment:
I'm starting to wonder if what you have with the `:` is actually better. I know I used the `-` in some of my tests but `:` looks pretty intuitive. What is the reason you changed it?

Reply:
`-` is used everywhere else in the codebase, such as this example in EPM's package registry: https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/server/services/epm/registry/index.ts#L65

I like `:` better too but I think consistency is the most important.

Reply:
I wonder if we mix two things here. The part you linked is the name of the `.zip` file I think. The reason we have `-` there is because it is pretty uncomment to have `:` in a file name. So it is `apache-1.2.3.zip` instead of `apache:1.2.3.zip` as file name. We also copied here what is used for the other stack downloads. I would think for configuration we could use the `:`. But as packages are not allowed to have any `-` in the name it is also something we could change later on and adjust all the places where it is used. So good to move forward with your change 👍